backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -18,29 +22,63 @@ class EakApplicationMutationService {
     fun addEakApplication(
         regionId: Int,
         application: Application,
+        project: String,
         dfe: DataFetchingEnvironment,
     ): Boolean {
         val context = dfe.getContext<GraphQLContext>()
+        val backendConfig = context.backendConfiguration
+        val projectConfig = backendConfig.projects.first { it.id == project }
```

Comment:
We should make sure that `regionId` stems from a region related to that project.

Also related: https://github.com/digitalfabrik/entitlementcard/issues/696